public/pages/workflow_detail/workflow_inputs/input_fields/model_field.tsx

```diff
@@ -119,11 +121,9 @@ export function ModelField(props: ModelFieldProps) {
               fullWidth={props.fullWidth}
               label={props.label || 'Model'}
               labelAppend={
-                <EuiText size="xs">
-                  <EuiLink href={ML_CHOOSE_MODEL_LINK} target="_blank">
-                    Learn more
-                  </EuiLink>
-                </EuiText>
+                props.modelCategory ? (
+                  <ModelInfoPopover modelCategory={props.modelCategory} />
+                ) : undefined
```

Comment:
Please default to the original learn more link if there is no model category, instead of setting to undefined.